sentry-opentelemetry/sentry-opentelemetry-core/src/test/kotlin/SentrySpanProcessorTest.kt

```diff
@@ -322,6 +332,40 @@ class SentrySpanProcessorTest {
         thenTransactionIsFinished()
     }
 
+    @Test
+    fun `links error to OTEL transaction`() {
+        fixture.setup()
+        val extractedContext = whenExtractingHeaders()
+
+        extractedContext.makeCurrent().use { _ ->
+            val otelSpan = givenSpanBuilder().startSpan()
+            thenTransactionIsStarted(otelSpan, isContinued = true)
+
+            otelSpan.makeCurrent().use { _ ->
+                val processedEvent = OpenTelemetryLinkErrorEventProcessor(fixture.hub).process(SentryEvent(), Hint())
+                val traceContext = processedEvent!!.contexts.trace!!
```

Comment:
Are we ok with `!!` in tests?

Reply:
I'd say yes, since it breaks the test if `null` comes along, so test still does what it's supposed to do. Also there's plenty of other `!!` in tests.